docs/snp.md

```diff
@@ -86,6 +94,10 @@ Attest the guest using the following command:
 ```
 ./snp.sh attest-guest
 ```
+On the Coconut-SVSM setup, use the following command:
+```
+./snp.sh --svsm attest-guest
+```
```

Comment:
```suggestion
Use the `--svsm` option in the above command to attest a Coconut-SVSM guest.
```

Reply:
You are asking remove command and add verbal explanation. I would like to keep command as well in documentation. This gives clear instruction on where they should add "--svsm" option. @ryansavino, any suggestions?